JustSaying.Extensions.DependencyInjection.Microsoft/JustSaying.Extensions.DependencyInjection.Microsoft.csproj

```diff
@@ -1,4 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\netfx.props" />
```

Comment:
Would these be better as `$(MSBuildThisFileDirectory)..\netfx.props` so it won't be affected by the current directory of the shell not being the solution root?

Reply:
Happy to change that if we prefer this approach over the myget assemblies approach

Reply:
@martincostello OK I updated the PR for that change